kubernetes_manager/models/kube.py

```diff
@@ -200,10 +201,12 @@ def get_obj(self):
         return client.V1PodTemplateSpec(
             metadata=client.V1ObjectMeta(name=self.slug, labels=self.labels, annotations=self.annotations),
             spec=client.V1PodSpec(
-                volumes=[self.volume.get_obj()] if self.volume is not None else None,
-                containers=[self.primary_container.get_obj(), self.secondary_container.get_obj()]
-                if self.secondary_container is not None
-                else [self.primary_container.get_obj()],
+                volumes=[
+                    v.get_obj() for v in self.volumes.all()
+                 ] if self.volumes is not None else None,
+                containers=[
```

Comment:
I'm assuming ordering isn't an issue and that names are autogenerated if they're not otherwise supplied (or are they a required field on the container object)?

Reply:
ordering doesnt matter, names are autogened using title/slug, title is required